test/integration/generated/lroRest/src/lRORestClient.ts

```diff
@@ -0,0 +1,1292 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Why is this filename seems a little weird ?

Reply:
I think Sarangan has a PR with some way to skip normalization. The naming logic tries to camel case, but it is not smart enough to identify acronyms.